webapp/channels/src/actions/notification_actions.jsx

```diff
@@ -111,16 +112,17 @@ export function sendDesktopNotification(post, msgProps) {
         const userStatus = getStatusForUserId(state, user.id);
         const member = getMyChannelMember(state, post.channel_id);
         const isCrtReply = isCollapsedThreadsEnabled(state) && post.root_id !== '';
+        const skipChecks = Boolean(post.props?.force_notification);
```

Comment:
Nitpick: I'd keep the name of `force_notification`, it is much more intentional than skipCheks